setup.py

```diff
@@ -62,10 +80,36 @@ def configuration(parent_package='',top_path=' '):
 
     return config
 
-try:
-    from holopy import __version__
-except ImportError:
-    __version__ = 'unknown'
+def _move_msvc_libs(mode='install'):
+    """ These dlls need to be moved if the fortran is complied in an environment
+    with MSVC 2015 as the C compiler.
+    """
+    package_dir = _get_holopy_install_dir(mode)
+    lib_dir = os.path.join(package_dir, '.libs')
+    sep = os.path.sep
+
+    tmatrix_libs = glob.glob(lib_dir + sep + 'libS.*.dll')
+    mie_libs = glob.glob(lib_dir + sep + 'libscsm*.dll')
+    mie_libs += glob.glob(lib_dir + sep + 'libmieang*.dll')
+    mie_libs += glob.glob(lib_dir + sep + 'libuts*.dll')
+
+    tmatrix_f_dir = os.path.join(package_dir, 'scattering', 'theory', 'tmatrix_f')
+    mie_f_dir = os.path.join(package_dir, 'scattering', 'theory', 'mie_f')
+
+    for dll in tmatrix_libs: shutil.move(dll, tmatrix_f_dir)
+    for dll in mie_libs: shutil.move(dll, mie_f_dir)
+    shutil.rmtree(lib_dir)
+
+def _get_holopy_install_dir(mode):
+    if mode == 'install':
+        sitepackages = list(site.getsitepackages())
```

Comment:
How robust is this way of finding where the package is installed?

Reply:
Well, it works on CI, so I stay a step above "works on my machine." 

For install mode AFIAK the only other directory that `site.getsitepackages()` returns is where the python executable is installed. 

For develop mode, there might be some case I'm missing.

Reply:
Refer to the official documentation for [python 3.6](https://docs.python.org/3.6/using/windows.html#finding-modules) and [python 3.7](https://docs.python.org/3.7/using/windows.html#finding-modules). On Windows, `site.getsitepackages()` returns two elements: the default python library, and the third-party library. 

According to the docs, the third party library is a sub-directory of the default library called "site-packages". The next line `path for path in site.getsitepackages() if 'site-packages' in path` selects this third party directory for the install location of holopy __if and only if the user has not manually edited their python path__.  Actually, now that I am re-reading it, line 105 is unneeded.

Therefore, unless the user has modified their python installation manually, this routine works on Windows. Also the routine only executes on Windows. I don't know why your site-package list includes libraries that are missing,.

Reply:
Also, I have no interest in trying to support *any* arbitrary python installation. If the user is that picky about how python is installed, they probably aren't on Windows. Even if they are, they probably know how to fix this. Otherwise, they should raise an issue and I'll address it.

Reply:
OK, sounds good @ralex0 . Can you just put a line or so of this discussion in a comment near the code, so if a problem does arise in the future there will be an easier time debugging this?

Otherwise I'm happy if you think this will work for every "generic Python user".

Reply:
Yeah this works in my fresh windows environments.